common/src/main/java/com/microsoft/identity/common/internal/broker/ipc/BrokerOperationBundle.java

```diff
@@ -68,6 +68,7 @@ public enum Operation {
         MSAL_GENERATE_SHR(API.GENERATE_SHR, BrokerAccountManagerOperation.GENERATE_SHR),
         BROKER_GET_KEY_FROM_INACTIVE_BROKER(null, null),
         BROKER_API_HELLO(API.BROKER_HELLO, null),
+        BROKER_API_UPLOAD_LOGS(API.BROKER_UPLOAD_LOGS, null),
```

Comment:
New enum constants are meant to be added at the end of the list because enums are ordinals under the hood and the values for existing constants mustn't be changed.